pkg/cmd/attestation/verification/policy.go

```diff
@@ -49,3 +49,57 @@ func (c EnforcementCriteria) Valid() error {
 	}
 	return nil
 }
+
+func (c EnforcementCriteria) BuildPolicyInformation() string {
+	template :=
+		`
+The following policy criteria will be enforced against all attestations:
+- Predicate type must match %s
+- Signing certificate's OIDC issuer must match %s
+- Artifact must be built in an organization whose URI is %s`
```

Comment:
tweak to language, tbh "source repository owner" is pretty understandable 🤔 / basically the name of the field

```suggestion
- Source Repository Owner URI must match %s`
```

Reply:
Thinking outloud, I guess I have a vibe we want to soften claims around the artifact per se. You could have done all sorts of shenanigans to generate this attestation, like curling the binary; the most we can say is this artifact was _seen_ inside this Actions workflow run.